R/create_extendr_package.R

```diff
@@ -0,0 +1,55 @@
+
+#' Create a project for R package development with Rust
```

Comment:
Consider rewriting `Create a package using extendr` or `Create package that uses Rust` or something. 

Generally, I dislike "for R package development." We're writing R so thats a given.